ParselyiOS/ParselyTracker.m

```diff
@@ -160,7 +163,10 @@ -(void)expelStoredEvent{
 }
 
 -(NSArray *)getStoredQueue{
-    return [[NSUserDefaults standardUserDefaults] objectForKey:storageKey];
+    if([[NSUserDefaults standardUserDefaults] objectForKey:storageKey] != nil) {
+        return [[NSUserDefaults standardUserDefaults] objectForKey:storageKey];
```

Comment:
Since we're dealing with the potential for multiple threads to be mutating this object at the same time, it might make sense to call `objectForKey` just once and use that value in both the conditional and the return. This negates the possibility that the object stored at that key will change between these two lines.